2-collectors/scala-stream-collector/examples/config.hocon.sample

```diff
@@ -195,17 +195,6 @@ collector {
     accessControlMaxAge = ${?COLLECTOR_CORS_ACCESS_CONTROL_MAX_AGE}
   }
 
-  # Configuration of prometheus http metrics
```

Comment:
I think we should keep this version (because it allows the values to be set as env vars) and delete the next section.

Reply:
I was talking with @chuwy and we decided that we would leave this version, because the vars aren't used anywhere.

Reply:
I would double-check that with Tech Ops. I think they do use the vars to setup collectors via the terraform stack.